chapter08/dtj/ex2

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+while (<>) {                   # take one input line at a time
+	chomp;
+	if (/(\ba\b)|\w+a\b/i) {
```

Comment:
This seems more complicated than necessary surly using * instead of + would be the same without needing the or?